konig-schemagen-maven-plugin/src/main/java/io/konig/maven/KonigSchemagenMojo.java

```diff
@@ -904,15 +906,18 @@ private void generateAmazonWebServices() throws IOException, ConfigurationExcept
 			if(tablesDir != null) {
 				SqlTableGenerator generator = new SqlTableGenerator();
 				DatasourceFileLocator sqlFileLocator = new DdlFileLocator(tablesDir);
-				AwsAuroraTableWriter awsAuror = new AwsAuroraTableWriter(tablesDir, generator,sqlFileLocator);
-			
+
+				File abbrevDir=new File(rdfSourceDir,"abbrev");
+				AwsAuroraTableWriter awsAuror = new AwsAuroraTableWriter(tablesDir, generator,sqlFileLocator,abbrevDir);
```

Comment:
Don't pass the abbrevDir location.  Instead pass an AbbreviationManager.  The AbbreviationManager should receive a graph in its constructor and lazy load the abbreviation descriptions into a map for each concept scheme.